index.bs

```diff
@@ -47,13 +48,54 @@ spec: WEBDRIVER; urlPrefix: https://w3c.github.io/webdriver/
 Introduction {#intro}
 =====================
 
+<em>This section is non-normative.</em>
+
 [[WEBDRIVER|WebDriver]] defines a protocol for introspection and
 remote control of user agents. This specification extends WebDriver by
 introducing bidirectional communication. In place of the strict
 command/response format of WebDriver, this permits events to stream
 from the user agent to the controlling software, better matching the
 evented nature of the browser DOM.
 
+Model {#model}
+==============
+
+This section defines the basic concepts of the WebDriver BiDi
+protocol. These terms are distinct from their representation at the
+<a href=#transport>transport</a> layer.
+
+Note: These definitions do not form any normative requirements, rather
+they are used elsewhere as part of such requirements.
+
+## Commands ## {#commands}
+
+A <dfn export>command</dfn> is an asynchronous operation, requested by
+the [=local end=] and run on the [=remote end=], resulting in either
+[=success=] (with a result) or [=error=] (with details) being returned
```

Comment:
This is a little confusing, because `success` and `error` are really spec-internal notions and you can't "send a success", you can just respond to the remote end steps returning a `success` by sending a response message over the wire and respond to them returning an `error` by sending an error message over the wire.

Reply:
You're right, I've pushed a change to not link the spec-internal notions.